template/build/webpack.prod.conf.js

```diff
@@ -41,6 +42,18 @@ var webpackConfig = merge(baseWebpackConfig, {
     new ExtractTextPlugin({
       filename: utils.assetsPath('css/[name].[contenthash].css')
     }),
+	new PurifyCSSPlugin({
+      // Give paths to parse for rules. These should be absolute! 
+        purifyOptions: {
+            whitelist: ['someclass']
```

Comment:
thanks for explanation. :)
this is just an example, and I prefer it being in a comment -- since a user may forget to change this.